mongodb/spring-cloud-starter-stream-source-mongodb/README.adoc

```diff
@@ -0,0 +1,26 @@
+//tag::ref-doc[]
+= MongoDB Source
+
+This source polls data from MongoDB.
+This source is fully based on the `MongoDataAutoConfiguration`, so refer to the
+http://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-nosql.html#boot-features-mongodb[Spring Boot MongoDB Support] for more
+information.
+
+== Options
+
+The **$$mongodb$$** $$source$$ has the following options:
+
+$$collection$$:: $$the collection within the database to target for the query$$ *($$String$$, no default, required)*
```

Comment:
Well, I'd say that "The MongoDB collection name." is strictly enough.
If that isn't obvious for end-user, let him go to the official MongoDB resources.

Sorry for nit-picking, but no reason to be verbose if a couple words can nail the subject.